protocol/communities_events_owner_without_community_key_test.go

```diff
@@ -40,6 +41,10 @@ func (s *OwnerWithoutCommunityKeyCommunityEventsSuite) GetEventSender() *Messeng
 	return s.ownerWithoutCommunityKey
 }
 
+func (s *OwnerWithoutCommunityKeyCommunityEventsSuite) GetEventSender2() *Messenger {
+	return s.ownerWithoutCommunityKey2
+}
+
```

Comment:
I wonder if it is possible to create this user in the common test scenario. The same as we did for the `User` in requests to join tests

Reply:
Like here
https://github.com/status-im/status-go/pull/3843/files#diff-2990ef21a6e7432ed6cdc95cc05b9f47439c44742fb1da050adb30d976ea1aa7R840-R845

Reply:
I will take a look at this!